### PR TITLE
[Reaf]: Updated homepage

### PR DESCRIPTION
**Pull request by Logan1x**
- updated homepage image
- card images updated
- OG Meta tags added
- mobile responsive
- alt tag added
- Signup Feat added
- fix test credentials bug

### CODE REVIEW

File: src/pages/home/Home.jsx [hunk 1]
```diff
@@ -14,47 +16,34 @@ export default function Home() {
...
-              src="https://picsum.photos/200/200"
-              alt=""
-            />
-            <div className="section-card-content">
-              <h3>
-                <Link to="/videos">Animation</Link>
-              </h3>
-            </div>
+            <Link to="/videos">
+              <img className="section-card-img" src={PopMusic} alt="" />
```
Comment (anuj-kumary) [inline src/pages/home/Home.jsx]: Don't leave **alt** attribute blank

**Review by anuj-kumary — APPROVED** [review]
- Clean UI
- Clean and understandable code
Good to merge

**Review by shrey27 — APPROVED** [review]
Overall code looks good to me
PR is ready to merge

**Comment by Logan1x** [comment]
Thanks Everyone, Merging now.

**Pull request merged**